src/vl-select.js

```diff
@@ -222,8 +242,15 @@ export class VlSelect extends vlFormValidationElement(nativeVlElement(HTMLSelect
    * @param {Object} params - object with callbackFn: function(select) with return value the items for `setChoices`
    * @fires VlSelect#VlSelectReady ready event wordt verstuurd wanneer veilige interactie met de webcomponent mogelijk is.
    */
+
   dress(params) {
     setTimeout(() => {
+      if (this.getAttribute('data-vl-search-placeholder')) {
+        this.searchPlaceholderTranslation = this.getAttribute('data-vl-search-placeholder');
```

Comment:
Duplicatie `this.getAttribute('data-vl-search-placeholder')` best een `searchPlaceholder` getter introduceren.